stack-orchestration/configuration/build/build-rp-stub-pipeline/parameters.json

```diff
@@ -13,7 +13,11 @@
   },
   {
     "ParameterKey": "GitHubRepositoryName",
-    "ParameterValue": "di-auth-stub-relying-party"
+    "ParameterValue": "none"
+  },
+  {
+    "ParameterKey": "GitHubRepositoryName",
+    "ParameterValue": "relying-party-stub"
```

Comment:
The variables should have different names :)

Reply:
🤦‍♀️